### DISCUSSION TITLE
Save Settings?

### DISCUSSION THREAD

**Discussion by 3Diva** [lllyasviel/Fooocus#617] [Q&A] [answered]
Is there a way to save user settings? Switching out the base model and changing all the settings every time I go to use Fooocus is extremely time consuming - and it seams to load the base model, refiner, and lora first anyway, even if I set the refiner and lora to "none" and change the checkpoint model before rendering the fist image. The log still says that it's loading the base model, refiner, and lora. Then it has to turn around and unload everything and load the checkpoint model that I selected. This seams to result in super long render times for the first image (note "Total time: 774.90 seconds" - that's about 13 minutes just to generate the first image):

![Screenshot 2023-10-09 213440](https://github.com/lllyasviel/Fooocus/assets/51475772/8d8449f2-216f-4cdf-84a9-9a6cf9eb12fe)

Is there a way to save the settings, so it that it remembers what model you want loaded and what settings you want to use? This would save so much time and be a HUGE help. I know that Fooocus-MRE has that option with a "settings" file that doesn't get changed when there's an update. And that's a MASSIVE HELP! Is there a way to save settings like that in Fooocus?

**Comment by lllyasviel** [comment, marked as answer]
I am considering similar things and will probaly solve this problem in the next update.

but spending 350 seconds to load model is probably because a known problem if users use HDD or when SSD does not have enough space

**Comment by lllyasviel** [comment]
fixed here

https://github.com/lllyasviel/Fooocus#customization

**Reply by 3Diva** [reply]
That was SO FAST! Thank you very much - I'm looking forward to testing it out!

**Reply by ksorah1** [reply]
in all others uis, params are saved on each picture and you just have to drag and drop them into uis to change parameters.

**Reply by julianCast** [reply]
Hi!So how does it work now? For example if I load Lora or new checkpoint they are reset again on each opening, was that changed? I don't see it.  Thanks!
